src/components/Button/Button.vue

```diff
@@ -595,6 +600,18 @@ export default {
 		}
 	}
 
+	// Tertiary on primary color (like the header)
+	&--vue-tertiary-on-primary {
```

Comment:
Should we just call it inverted colours  or something like that? I can imagine we could use something similar in talk's call view

Reply:
No that's not how it works. In Talk we would always need white or something alike, not depening on the primary color nor the background.